architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,214 @@
+----
+
+AWS User::
+The AWS Service Broker requires an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html[AWS Identity Access Management (IAM) user] and the associated link:http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[IAM keys] to provision AWS services. The user needs *Programmatic access* and *AWS Management Console* access types and add the two policies you created.
+
+IAM Role::
+The AWS Service Broker requires an IAM Role to access link:https://aws.amazon.com/cloudformation/[AWS CloudFormation] when provisioning services. 
+You must give CloudFormation full permission to the role. 
+
+Use the the AWS Management Web Console to create this role for the *CloudFormation* entity and assign *AdministratorAccess* to the role.
+
+Amazon Resource Name::
+After creating the role, locate the Amazon Resource Name (ARN) on the role access summary page using the AWS console. You need the role ARN during the deployment process.
+
+The role ARN has the following format:
+
+[source,bash]
+----
+arn:aws:iam::47458525407874:role/CloudFormation
+----
+
+[NOTE]
+====
+Optionally, the AWS Service Broker can link:https://github.com/awslabs/aws-servicebroker/tree/master/docs#managing-resources-via-assumed-role[manage multiple clusters across AWS accounts using a single role].
+====
+
+AWS Command Line Interface::
+If you want to use the command line to work with AWS, you must install the link:https://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS Command Line Interface], which requires
+link:https://developers.redhat.com/blog/2018/08/13/install-python3-rhel/[Python], and link:https://access.redhat.com/solutions/1519803[pip].
+
+DynamoDB Table::
+You need to create an link:https://console.aws.amazon.com/dynamodb/home?region=us-east-1#gettingStarted:[Amazon DynamoDB] table for persistent storage.
+
+.Procedure
+
+Run the following command in the AWS CLI to create the table:
+
+[source,bash]
+----
+aws dynamodb create-table --attribute-definitions \
+AttributeName=id,AttributeType=S AttributeName=userid,AttributeType=S \
+AttributeName=type,AttributeType=S --key-schema AttributeName=id,KeyType=HASH \
+AttributeName=userid,KeyType=RANGE --global-secondary-indexes \
+'IndexName=type-userid-index,KeySchema=[{AttributeName=type,KeyType=HASH},{AttributeName=userid,KeyType=RANGE}],Projection={ProjectionType=INCLUDE,NonKeyAttributes=[id,userid,type,locked]},ProvisionedThroughput={ReadCapacityUnits=5,WriteCapacityUnits=5}' \
+--provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
+--region us-east-1 --table-name awssb <1>
+----
+
+<1> Change the `--region` and `--table-name` parameters as needed. 
+
+The command output reports that the table is created or there is an error. The output for a successful creation includes:
+
+----
+"CreationDateTime": 1539185077.534
+----
+
+You can also use the DynamoDB dashboard in AWS to see the table. 
+
+[[aws-getting-started-before-policies]]
```

Comment:
Wrong heading style for modules. Also, you aren't supposed to have more than one non-discrete heading in a proc module.